tools/generation/test/fixtures/normal/info-multiline-folding.template

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+name: First template name
+path: normal/info-multiline-folding-
+es6id: 1.2.3
+info: |
+    This is an "info" field with an
+
+    empty line
+---*/
```

Comment:
how about leaving a trailing whitespace in the frontmatter to see it is removed and another in the code where it probably should be kept?